src/redprl/refiner.fun

```diff
@@ -338,6 +338,14 @@ struct
     fun ReducePart sign = SequentReducePart sign
   end
 
+  fun LabelGoal (lbl : string) : rule = 
+    fn alpha => fn H >> jdg => 
+      let
+        val (goal, hole) = makeGoal [lbl] @@ H >> jdg
+      in
+        |>: goal #> (H, hole)
+      end  
```

Comment:
I hope this works for `MATCH` and `MATCH_RECORD` as well. Is there a reason to limit it to sequents?

Reply:
thanks! I've fixed it now.